### ISSUE TITLE
Add posibility to log messages with different log levels

### ISSUE THREAD

**Issue by qarmin** [stevenliebregt/fun_time#5]
For now only this two are allowed
```
#[fun_time(message = "ABC", reporting = "log")] # Prints with info!
#[fun_time(message = "ABC", reporting = "println")] # Prints with println!
```
but I want to print such messages with debug! macro instead

**Comment by stevenliebregt** [comment]
I will have a look at that

**Comment by stevenliebregt** [comment]
Fixed in `0.3.2` https://github.com/stevenliebregt/fun_time/releases/tag/0.3.2
